tests/golos.ctrl_tests.cpp

```diff
@@ -458,5 +460,53 @@ BOOST_FIXTURE_TEST_CASE(update_auths, golos_ctrl_tester) try {
     BOOST_CHECK(top_witnesses["witnesses"].as<std::vector<name>>().size() == wtns.size());
 } FC_LOG_AND_RETHROW()
 
+BOOST_FIXTURE_TEST_CASE(payments_top_witness, golos_ctrl_tester) try {
+    prepare_balances();
```

Comment:
It is very useful to add a brief description of the actions that are done in the test. This helps to quickly understand what is being checked.
You can see it in https://github.com/GolosChain/golos.contracts/pull/718/files#diff-fb14cf9885b6f0b4d694ac942805c21eR174.